packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -31,24 +27,11 @@ export async function getWorkspaceOrgType(
     defaultUsernameOrAlias
   );
   const isUsername = email ? true : false;
-
```

Comment:
We can now remove the retry code since the org create command will now set the correct context after it finishes executing.